cmd/server/main.go

```diff
@@ -114,3 +115,12 @@ func buildCLI() *cli.App {
 	}
 	return app
 }
+
+// For backward compatiblity to support old flag format (i.e. `--services=frontend,history,matching`).
+func splitServices(services []string) []string {
+	var result []string
+	for _, service := range services {
+		result = append(result, strings.Split(service, ",")...)
```

Comment:
The original code also trimmed spaces before splitting:
```go
val := strings.TrimSpace(c.String("services"))
```

Reply:
It doesn't make sense for me actually. Also `cli` lib should handle everything that shell did not handle itself.